src/axom/primal/tests/primal_bezier_curve.cpp

```diff
@@ -142,6 +142,37 @@ TEST(primal_beziercurve, evaluate)
   }
 }
 
+//------------------------------------------------------------------------------
+TEST(primal_beziercurve_, point_derivatives)
+{
+  SLIC_INFO("Testing Bezier derivative calculations on a low-order curve");
+
+  const int DIM = 3;
+  using CoordType = double;
+  using PointType = primal::Point<CoordType, DIM>;
+  using VectorType = primal::Vector<CoordType, DIM>;
+  using BezierCurveType = primal::BezierCurve<CoordType, DIM>;
+
+  PointType data1[1] = {PointType {0.6, 1.2, 1.0}};
+  PointType data2[2] = {PointType {0.6, 1.2, 1.0}, PointType {1.3, 1.6, 1.8}};
+
+  BezierCurveType pointCurve(data1, 0);
+  BezierCurveType lineCurve(data2, 1);
```

Comment:
:+1 Thanks for adding tests for order 0 and 1!

Reply:
Should we also explicitly test the calls to `evaluate_first_derivative` and `evaluate_second_derivative` for order 0 and 1? Or is it sufficient for them to be tested implicitly through `dt()` and `dtdt()`?

Reply:
For sure! As it was, requesting `.dt` on a 0th order curve would cause an error, and that didn't feel right to me. 

And yes, some of the tests I want to add will flex the capability of `evaluate_first/second_derivatives` on 0th and first order curves. The methods called are slightly different than the `dt` and `dtdt`.